RecoCTPPS/TotemRPLocal/python/totemRPLocalReconstruction_cff.py

```diff
@@ -0,0 +1,24 @@
+import FWCore.ParameterSet.Config as cms
+
+# geometry
+from Geometry.VeryForwardGeometry.geometryRP_cfi import *
+XMLIdealGeometryESSource.geomXMLFiles.append("Geometry/VeryForwardData/data/RP_Garage/RP_Dist_Beam_Cent.xml")
```

Comment:
makes sense to me, at least as a minimal solution.
Database will eventually be better.
@ianna , what do you think?